sdk/bpf/c/inc/sol/log.h

```diff
@@ -40,6 +40,19 @@ static void sol_log_array(const uint8_t *array, int len) {
   }
 }
 
+/**
+ * A single data field for sol_log_data
+ */
+typedef struct {
+  const uint8_t *data;
+  uint64_t len;
+} SolLogDataField;
```

Comment:
Any reason not to use `SolBytes` defined in `sdk/bpf/c/inc/sol/types.h`?

Reply:
Good point, I've fixed that.